packages/grafana-ui/src/components/PieChart/PieChart.tsx

```diff
@@ -19,7 +19,7 @@ export enum PieChartLabels {
   Percent = 'percent',
 }
 
-export enum LegendColumns {
+export enum PieChartLegendValues {
```

Comment:
renamed this and made it exported from package, since all components in grafana-ui share one namespace, need to be very explicit with the names we export so they are not too generic etc.